setup_wordpress.sh

```diff
@@ -1,3 +1,91 @@
 #!/bin/bash
 
 echo 'This script should install and setup Wordpress'
+
+
+# Variables needed for configurations these can be edited to anything you want
+Dbname="wordpress"
+Dbuser="wpuser"
+Dbpass="Devops101"
+Wppass="!2three456."
+
+###############
+# nginx, mysql, WordPress.
+# WordPress should have already configured with theme twentynineteen and with created user admin with password !2three456..
+#####################
+
+# install packages
+sudo apt update
+sudo apt-get upgrade -y
+sudo apt install nginx -y
+sudo apt install mysql-server -y 
+sudo apt install php7.4 php7.4-fpm php7.4-mysql php7.4-curl php7.4-gd php7.4-intl php7.4-mbstring php7.4-soap php7.4-xml php7.4-xmlrpc php7.4-zip -y
+
+
+#configure firewall nginx
+sudo ufw enable
+sudo ufw allow 'Nginx HTTP'
+
+# install and configure mysql-server
+sudo mysql -e "CREATE DATABASE $Dbname DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;"
+sudo mysql -e "CREATE USER '$Dbuser'@'localhost' IDENTIFIED BY '$Dbpass';"
+sudo mysql -e "GRANT ALL ON $Dbname.* TO '$Dbuser'@'localhost';"
+sudo mysql -e "FLUSH PRIVILEGES;"
+
+# Permissions configurations for wordpress
+cd /tmp
+sudo wget https://wordpress.org/latest.tar.gz
+tar xzvf latest.tar.gz
+sudo cp /tmp/wordpress/wp-config-sample.php /tmp/wordpress/wp-config.php
```

Comment:
why is `sudo` needed here?

Reply:
I have a thing for always using sudo with my commands. I do not know if it is a bad or good habit.

Reply:
I would recommend using `sudo` only when you need special permissions.

Reply:
oke thanks for the tip i will keep it in mind.